lib/eldap/src/eldap.erl

```diff
@@ -301,14 +384,20 @@ info(Handle) when is_pid(Handle) ->
 %%%  Dn      -  The name of the entry to bind as
 %%%  Passwd  -  The password to be used
 %%%
-%%%  Returns: ok | {error, Error}
+%%%  Returns: - return_value()
 %%% --------------------------------------------------------------------
```

Comment:
We agreed return_value() should not be used. And even if you took it out of all the specs it is still present in the documentation in many places. In this place you could just remove it, as it is now a redundant code comment.
But it is used in the documentation text and it makes the function summarizes look bad. 

First looks like:
`add(Handle, Dn, [Attribute]) -> return_value()`

I think it should say something like : "Requests an addition on the entry {Dn, Attributes} to the LDAP server."

Reply:
This goes for all the functions.  You can take some inspiration from 
 https://datatracker.ietf.org/doc/html/rfc4511#section-4.7 to figure out better descriptions. The current ones are really sparse.